GSA/digitalgov.gov
# Header secondary nav & Footer: Link update

## Summary

This PR will update links above the search bar along with adding a new link to the footer menu. 

### Preview

Header before/after
<img width="393" alt="Screenshot 2023-10-03 at 9 59 10 AM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/82c77f81-5155-4b98-aca9-499670b5faf1"> <img width="380" alt="Screenshot 2023-10-03 at 10 01 04 AM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/576867f9-1637-4949-9cee-f01d99a7a30c">

Footer before/after
<img width="157" alt="Screenshot 2023-10-03 at 9 59 16 AM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/1ceffe66-7a43-47e6-9a7c-ff157102534e"> <img width="146" alt="Screenshot 2023-10-03 at 10 01 08 AM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/5e433095-61a9-4415-af4e-21b07bd513fd">




[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-link-update/)



### Solution

- Moved Contact link from Secondary nav to Footer.
- Changed language in header subscribe link.

### How To Test

1. Visit [Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-link-update/)
2. Verify that links above the search are updated and the footer menu has been updated as well (see below for requirements).

### What to test

- [ ] About is removed from secondary nav & now appears in footer.
- [ ] Remove Write for us
- [ ] Change `Subscribe` to Subscribe to our newsletter or something like that
- [ ] Contact link is moved to the footer under the `About` column

---

### Dev Checklist
